_redirects

```diff
@@ -13,6 +13,7 @@
 /hpo/                  https://docs.rapids.ai/deployment/stable/examples/index.html
 /hpo.html              https://docs.rapids.ai/deployment/stable/examples/index.html
 /introgtc2023/         https://github.com/shwina/rapids-tutorial-gtc-2023
+/nx-cugraph            https://github.com/rapidsai/cugraph/tree/HEAD/python/nx-cugraph/README.md
```

Comment:
```suggestion
/nx-cugraph/            https://github.com/rapidsai/cugraph/tree/HEAD/python/nx-cugraph/README.md
```

We have a higher-priority redirect that adds a trailing slash to pages that are missing it. Therefore this source column needs a trailing slash.